### DISCUSSION TITLE
Coqui Xtts-v2

### DISCUSSION THREAD

**Discussion by mizbv** [coqui-ai/TTS#3715] [General Q&A]
i was just reading Coqui Xtts-v2 documentation but i cant seem to find the number off parameters

**Comment by MethanJess** [comment]
May I ask what are you trying to achieve? or what you mean by "The number off parameters"?

**Reply by mizbv** [reply]
so i'am working on my resume and i wanted to do a benchmarking comparaison between the models i used
